src/main/java/de/tum/in/www1/artemis/domain/SelfLearningFeedbackRequest.java

```diff
@@ -0,0 +1,116 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.time.ZonedDateTime;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonIncludeProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonView;
+
+import de.tum.in.www1.artemis.domain.participation.Participation;
+import de.tum.in.www1.artemis.domain.view.QuizView;
+
+/**
+ * A Self Learning Request.
+ */
+@Entity
+@Table(name = "self_learning_feedback_request")
+@EntityListeners({ AuditingEntityListener.class })
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class SelfLearningFeedbackRequest extends DomainObject {
+
+    @ManyToOne
+    @JsonView(QuizView.Before.class)
+    @JsonIgnoreProperties({ "results", "submissions", "selfLearningFeedbackRequests" })
+    private Participation participation;
```

Comment:
This can easily lead to inconsistencies when either the participation or submission is not set. You don't need to load all submissions either. Only the one submission linked to the FeedbackRequest and its participation are enough. This can also be loaded easily in one single database query so the performance is not affected that much